model_analyzer/config/input/config_defaults.py

```diff
@@ -41,8 +41,8 @@
 DEFAULT_CLIENT_PROTOCOL = 'grpc'
 DEFAULT_RUN_CONFIG_MAX_CONCURRENCY = 1024
 DEFAULT_RUN_CONFIG_MIN_CONCURRENCY = 1
-DEFAULT_RUN_CONFIG_MAX_REQUEST_RATE_RANGE = 1024
-DEFAULT_RUN_CONFIG_MIN_REQUEST_RATE_RANGE = 1
+DEFAULT_RUN_CONFIG_MAX_REQUEST_RATE_RANGE = 8192
+DEFAULT_RUN_CONFIG_MIN_REQUEST_RATE_RANGE = 1024
```

Comment:
why such a high default? Aren't there some slow models that can't hit anywhere near this?

Reply:
Lowered the default